modyn/supervisor/internal/eval/strategies/between_two_trainings.py

```diff
@@ -1,15 +1,22 @@
-from typing import Iterable, Optional
```

Comment:
note: the file is named between_two_trainings but the class between two triggers

Reply:
Thanks!